lua/nvim-tree/explorer/explore.lua

```diff
@@ -81,14 +81,15 @@ function M.explore(node, status)
     return ns
   end
 
-  sorters.sort(node.nodes)
+  M.sorters:sort(node.nodes)
   live_filter.apply_filter(node)
 
   log.profile_end(profile)
   return node.nodes
 end
 
 function M.setup(opts)
+  M.sorters = Sorters:new(opts)
```

Comment:
This will have the lifetime of nvim-tree, as it's called once during setup.

We'll need to move this into the Explore constructor so that it has the lifecycle of just the Explore instance.
See init.lua:  `function Explorer.new(path)`